index.js

```diff
@@ -45,6 +46,7 @@ module.exports.makeApp = function () {
   const themeName = config.get('THEME')
   if (themeName) {
     app.use('/static', express.static(path.join(__dirname, `/${themeDir}/${themeName}/public`)))
+    // app.use('/robots.txt, 
```

Comment:
@gavram can we remove this line?

Reply:
@anuveyatsu Yes, I can. Sorry, it's my mistake.